imap_data_access/file_validation.py

```diff
@@ -204,7 +205,11 @@ def generate_from_inputs(
             extension = "pkts"
         time_field = start_time
         if repointing:
```

Comment:
I realize this was already there before, but we should change it to `is not None` so we can handle 0 too.

Reply:
@maxinelasp can you update this too?